artscraper/artist_links.py

```diff
@@ -0,0 +1,38 @@
+from selenium import webdriver
+from googleart import *
```

Comment:
You should only import the functions/classes that you need.

Reply:
I can't import from googleart anymore after fiddling with the __init.py__ file. Now I do:

`from artscraper import random_wait_time`

However, I don't understand why I can't import from googleart anymore.